docs/source/tutorials/configurable.rst

```diff
@@ -0,0 +1,108 @@
+instance of Sopel yourself, you probably had to open and edit its configuration
+file.
+
+Usually located in your home directory, under the ``.sopel/`` folder, the
+configuration file is an INI file with sections defined by Sopel's core and by
+plugins. In this tutorial, let's see how to declare and use a configuration
+section dedicated to your plugin.
+
+
+Declaring your configuration
+============================
+
+To declare a configuration section, you must first create a subclass of
+:class:`~sopel.config.types.StaticSection`, and define attributes::
+
+    from sopel.config import types
+
+    class MyPluginSection(types.StaticSection):
+        fruits = types.ListAttribute('fruits')
+
+
+Telling Sopel about it
+======================
+
+Now, having a class in your plugin doesn't achieve much: you need to tell the
+bot about it. The best place to do so is in the :func:`setup` function
+hook of your plugin::
+
+    def setup(bot):
+        bot.settings.define_section('myplugin', MyPluginSection)
+
+This way, you tell Sopel that the ``[myplugin]`` section is used by your
+plugin, and to parse this section Sopel must use your class, i.e.
+``MyPluginSection``.
+
+
+Using your section
+==================
+
+In your configuration file, you can add the following lines:
+
+.. code-block:: ini
+
+    [myplugin]
+    fruits =
+        banana
+        apple
+        peach
+        strawberry
+
+And Sopel will expose that for you through ``bot.settings``. For example, you
+can write this command::
+
+    import random
+
+    @plugin.command('fruits')
+    def fruits(bot, trigger):
+        fruit = random.choice(bot.settings.myplugin.fruits)
+        bot.say(f'I want a {fruit}!')
+
+And whenever someone triggers this command, the bot will say that it wants one
+of the configured fruits. If you want to list 50 fruits or only 2 is up to you,
+and to the bot owners who will install your plugin.
+
+
+Putting everything together
+===========================
+
+We can combine all of this into one plugin file, located at the same place as
+before (``~/.sopel/plugins/myplugin.py``, assuming the default location)::
+
+    import random
+    from sopel.config import types
+
+
+    class MyPluginSection(types.StaticSection):
+        """Declaration of your plugin's configuration."""
+        fruits = types.ListAttribute('fruits')
+
+
+    def setup(bot):
+        """Telling the bot about the plugin's configuration."""
+        bot.settings.define_section('myplugin', MyPluginSection)
+
+
+    @plugin.command('fruits')
+    def fruits(bot, trigger):
+        """Using the plugin's configuration in our command."""
+        fruit = random.choice(bot.settings.myplugin.fruits)
+        bot.say(f'I want a {fruit}!')
+
+As you can see, there are several steps when it comes to configuration. Sopel
+tries to make it as straightforward and flexible as possible for you to declare
+and to setup your plugin configuration, and it offers various possibilities.
+
```

Comment:
I think this paragraph can be dropped entirely, especially since we're about to refer more directly to the `various possibilities` by linking to more thorough documentation :grin: 

```suggestion
```

Reply:
No, it is important to remind the reader that there are multiple steps in the process. It is a warning that, hey, they **must** pay attention to all the requirements. The last bit about "offers various possibilities" could be dropped, reading it now it feels unnecessary.